File: C++/1013.cpp
#include <stdio.h>
#include <math.h>
 
int main() {
 
  int A,B,C,AB,ABC;
  scanf("%d %d %d",&A,&B,&C);
  
  AB = ((A+B+abs(A-B))/2);
  ABC = ((AB+C+abs(AB-C))/2);
  
  printf("%i eh o maior\n", ABC);
  
  
  
  
  return 0;

}

File: Python/1013.py
a, b, c = input().split()
a = int(a)
b = int(b)
c = int(c)
maior = a
if (a >= b) and (a >= c):
    print(f'{a} eh o maior')
elif (b >= a) and (b >= c):
    print(f'{b} eh o maior')
elif (c >= a) and (c >= b):
    print(f'{c} eh o maior')
